src/transformers/astro.test.ts

```diff
@@ -0,0 +1,74 @@
+import { assertEquals } from "https://deno.land/std@0.172.0/testing/asserts.ts";
+
+import { ParsedUrl } from "../types.ts";
+import { AstroParams, parse, transform } from "./astro.ts";
+
+const img =
+  "https://images.ctfassets.net/aaaa/xxxx/yyyy/how-to-wow-a-customer.jpg";
+
+Deno.test("astro parser", () => {
+  const parsed = parse(img);
+  const expected: ParsedUrl<AstroParams> = {
+    base: "/_image?href=https%3A%2F%2Fimages.ctfassets.net%2Faaaa%2Fxxxx%2Fyyyy%2Fhow-to-wow-a-customer.jpg",
+    params: {
+      "href": "https%3A%2F%2Fimages.ctfassets.net%2Faaaa%2Fxxxx%2Fyyyy%2Fhow-to-wow-a-customer.jpg"
+    },
+    cdn: "astro"
+  }
+  assertEquals(JSON.stringify(parsed), JSON.stringify(expected));
+});
+
+Deno.test("astro parser endpoint", () => {
+  const parsed = parse("/_image?href=https%3A%2F%2Fimages.ctfassets.net%2Faaaa%2Fxxxx%2Fyyyy%2Fhow-to-wow-a-customer.jpg");
+  const expected: ParsedUrl<AstroParams> = {
+    base: "/_image?href=https%3A%2F%2Fimages.ctfassets.net%2Faaaa%2Fxxxx%2Fyyyy%2Fhow-to-wow-a-customer.jpg",
+    params: {
+      "href": "https%3A%2F%2Fimages.ctfassets.net%2Faaaa%2Fxxxx%2Fyyyy%2Fhow-to-wow-a-customer.jpg"
+    },
+    cdn: "astro"
+  }
+  assertEquals(JSON.stringify(parsed), JSON.stringify(expected));
+});
+
+
+Deno.test("astro", async (t) => {
+  await t.step("should format a URL", () => {
```

Comment:
Can you add a test for relative URLs please

Reply:
Sure 👍 I added a test for relative URLs